src/script.js

```diff
@@ -1 +1,65 @@
+/* eslint-disable no-const-assign */
 import './style.css';
+
+// Store your API key and game ID in variables
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] I noticed that you disabled **Eslint** . Please remove the **eslint disable** statement and try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1: